app/models/miq_queue.rb

```diff
@@ -174,6 +174,12 @@ def self.get(options = {})
     result
   end
 
+  # This are the queue calls related to worker management which
+  # might not be needed once we use kubernetes for worker/pod management
+  def self.put_deprecated(*args)
```

Comment:
@jrafanie Its just an alias for now, it still calls the MiqQueue.put but we think these might go away at some point once we move to Kubernetes